core/src/main/java/com/redhat/runtimes/inventory/auth/principal/IllegalIdentityHeaderException.java

```diff
@@ -0,0 +1,10 @@
+/* Copyright (C) Red Hat 2023 */
+package com.redhat.runtimes.inventory.auth.principal;
+
+// Derived from notifications-backend
+public class IllegalIdentityHeaderException extends Exception {
+
```

Comment:
Custom exception classes should define all constructors.

Reply:
Done